### ISSUE TITLE
Invalid Solutions

### ISSUE THREAD

**Issue by gitcsec** [nginnever/zogminer#36]
Hi,

The GPU miner is running for quite some time without crashing. Nonetheless, I'm getting a lot of the following messages:
 
Solution found; Submitting to zec.suprnova.cc...
  000000000000000000000000000002ffa1eeff0f00000000000000000fffeea2
Invalid solution: invalid collision length between StepRows
Invalid solution: invalid collision length between StepRows
[WARN] FAILURE: Miner gave incorrect result!

Might be a bug?

**Comment by omaralvarez** [comment]
No, with the new kernel there is still no stratum support. I'm working on it. I am almost there, expect a commit soon.

**Comment by iamniks** [comment]
Any news on this?

**Comment by omaralvarez** [comment]
Yep, just fixed this. But we need a special port on suprnova for my fix to work. We need a special nonce, due to the new kernel.
